manifests/k/Kiwix/Wikivoyage/Electron/2.2.7-E/Kiwix.Wikivoyage.Electron.locale.en-GB.yaml

```
@@ -0,0 +1,38 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Kiwix.Wikivoyage.Electron
+PackageVersion: 2.2.7-E
+PackageLocale: en-GB
+Publisher: Kiwix
+PublisherUrl: https://github.com/kiwix/kiwix-js-windows
+PublisherSupportUrl: https://github.com/kiwix/kiwix-js-windows/issues
+Author: Kiwix
+PackageName: Wikivoyage by Kiwix Electron Edition
```

```suggestion
PackageName: Wikivoyage by Kiwix
```
To match Control Panel

---

@mdanish-kh Thank you for the suggestion. However, for the three apps I publish, Kiwix JS, WikiMed and Wikivoyage, there are two editions. One is called, e.g. "WikiMed by Kiwix" (this is the UWP app), and the other is called "WikiMed by Kiwix Electron Edition". I need the user to be able to distinguish between them in case they have both installed, hence they need different titles. I could shorten this one to "Wikivoyage by Kiwix Electron" (without "Edition") if it seems better, but I do need a different name than just "Wikivoyage by Kiwix".

---

@Jaifroid Thanks for the reply. I made the suggestion keeping in mind that WinGet upgrade logic compares values written to Registry and the metadata written in the manifest. 

For `exe` installers, PackageVersion and PackageName fields are more important in finding a match where they should match the values in Control Panel / Registry. But AFAIK, WinGet will try to regex match PackageName, so I believe having `Wikivoyage by Kiwix Electron Edition` would still work. Ofcourse, we can use `AppsAndFeatureEntries` in the manifest and explicitly provide DisplayName to force match an upgrade but these entries are not automation friendly and would need to be updated manually for each version.

If you haven't had any issues upgrading this package, then there's no need for me to be picky here.

---

OK, thanks for the advice. I'll keep it under observation for now, and change/shorten the name if any issues arise with upgrades.